src/watchlists/EditorMode.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { FontAwesome5 } from '@expo/vector-icons';
+import { View, Text, TouchableHighlight } from 'react-native';
+import PropTypes from 'prop-types';
+import styles from './styles';
+
+const editorMode = ({ checkedItems, isEdit, toggle }) => {
```

Comment:
Lets call this component `EditorControls`, `EditControls`, `EditorFooter`, `EditorButtons`, etc...